docs/conf.py

```diff
@@ -55,7 +55,7 @@
 
 # General information about the project.
 project = 'NI Modular Instruments Python API'
-copyright = '2017-2019, National Instruments'
+copyright = '2017-2020, National Instruments'
```

Comment:
Should we programmatically fill this out? If so, I can open an issue for it.

Reply:
It is programmatically set along with the version. Look at build/templates/conf.py.mako